### DISCUSSION TITLE
Language change

### DISCUSSION THREAD

**Discussion by elpaul** [mampfes/hacs_waste_collection_schedule#3274] [Q&A]
Good evening
I have the latest version of the plugin installed, I installed it yesterday via UI
How do I change the selected words to be displayed in the same language as Home Assistant? I use Polish, and here I have English.
![Zrzut ekranu 2024-12-24 161737](https://github.com/user-attachments/assets/e69b67dd-5c1b-45ce-b0e7-7d166972195b)

**Comment by 5ila5** [comment]
You currently cannot translate the lower two values you circled.

The first (and second) value is generated using the value template of the sensor. You can modify this by clicking `customize` next to your source (setting -> devices and scenes -> Waste Collection Schedule -> customize (next to your source)) and selecting the sensors you want to change -> submit -> change value template -> submit.
